web-console/src/lib/functions/felderaRelation.ts

```diff
@@ -0,0 +1,28 @@
+import { Field, Relation } from '$lib/services/manager'
+import invariant from 'tiny-invariant'
+
+export type CaseDependentName = {
+  case_sensitive?: boolean | undefined
+  name: string
+}
+
+export const quotifyRelationName = (relation: Relation) =>
```

Comment:
I think the verb is "quote".
So "quoteRelationName"

Reply:
This should be a method of the Relation class.

Reply:
Well the name is not necessarily gets quoted, it depends on whether it is case sensitive or not; so I am renaming `quotifyRelationName` and `quotifyFieldName` to `getCaseIndependentName`, and `quotedRelationName` to `caseIndependentName`